CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```

Comment:
Breaking change. We should release `1.0` ;)

Reply:
:) I suppose not having to do that is the reason for the Hex badge in README being orange.